stylesheets/mainpagestyle.css

```diff
@@ -116,25 +119,26 @@ body {
 }
 
 .program-icon {
-  width: 20%;
+  width: 15%;
   padding: 1%;
   object-fit: scale-down;
 }
 
 .program-title {
-  width: 25%;
+  width: 30%;
   color: #ec5242;
   font-size: 1.1em;
   font-weight: 800;
+  text-align: center;
 }
 
 .program-description {
-  padding: 0 2%;
+  padding: 0 1%;
   color: #fff;
-  width: 55%;
+  width: 50%;
   text-align: left;
-  font-size: 0.8em;
-  font-weight: 500;
+  font-size: 0.9em;
+  font-weight: 700;
```

Comment:
- [ ] [OPTIONAL] Good job implemeting the changes requested by the previous reviewer, however the cards in mobile view still look a bit busy, I think we can improve it by setting justify-content on .program-title to space between and removing the bold property for font weight and by reducing the width property to say 40-45% , how it looks now :point_down: 
![image](https://user-images.githubusercontent.com/97521557/210400524-1245baba-b979-48e7-b10d-095b0f829d07.png)

how it could be :point_down:
 
![image](https://user-images.githubusercontent.com/97521557/210400822-39f44e19-9f54-455c-acab-f42b0295fd50.png)

Reply:
@PinkMoon25 hi Tushar, thanks for this feedback. I ll implement the suggestion.